pkg/ipmi/ipmi.go

```diff
@@ -112,11 +113,7 @@ func (i *IPMI) ShutoffWatchdog() error {
 	data[5] = 0x0b // countdown msb - 5 mins
 
 	_, err := i.SendRecv(_IPMI_NETFN_APP, BMC_SET_WATCHDOG_TIMER, data[:6])
-	if err != nil {
```

Comment:
return i.SendRecv(...)?

Reply:
Will not work i.SendRecv returns ([]byte, error) but LogSystemEvent return error.

Reply:
oops, what was I thinking.